invenio_app_ils/ext.py

```diff
@@ -256,13 +257,15 @@ class InvenioAppIlsREST(InvenioAppIls):
     def init_app(self, app):
         """Flask application initialization."""
         super(InvenioAppIlsREST, self).init_app(app)
-        self._register_signals()
+        self._register_signals(app)
         app.errorhandler(RESTException)(handle_rest_exceptions)
 
-    def _register_signals(self):
+    def _register_signals(self, app):
         """Register signals."""
         register_circulation_signals()
         register_files_signals()
+        if app.config["EXTEND_LOANS_LOCATION_UPDATED"]:
```

Comment:
For the next PR, not this one, I would propose to remove the config var `EXTEND_LOANS_LOCATION_UPDATED` from the `config.py` since it is used only for tests and change this to:
```suggestion
        if app.config.get("EXTEND_LOANS_LOCATION_UPDATED", True):
```